app/Policies/InvoicePolicy.php

```diff
@@ -10,6 +10,17 @@ class InvoicePolicy
 {
     use HandlesAuthorization;
 
+    public function before(User $user)
+    {
+        if ($user->isAdmin()) {
+            return true;
+        }
+
+        // Don't write it like below. Write it like above.
+        // Otherwise, we will have broken the authorization for non-admins
+        // return $user->isAdmin();
```

Comment:
If `before` returns a non-null result, that result will be considered the result of the authorization check. If null is returned, the authorization check will fall through to the policy method.